force-app/main/default/triggers/MeetupRegistrationTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger MeetupRegistrationTrigger on MeetupRegistration__c (before insert) {
+    if (Trigger.isBefore && Trigger.isInsert) {
```

Comment:
Typically I would use some kind of 1-line trigger framework. I'm partial to https://github.com/kevinohara80/sfdc-trigger-framework